rollbar-java/src/main/java/com/rollbar/notifier/Rollbar.java

```diff
@@ -1,4 +1,633 @@
+   *
+   * @param error the error.
+   * @param description human readable description of error.
+   * @param level the level.
+   */
+  public void log(Throwable error, String description, Level level) {
+    log(error, null, description, level);
+  }
+
+  /**
+   * Record an error with custom parameters and human readable description at the default level
+   * returned by {@link Rollbar#level}.
+   *
+   * @param error the error.
+   * @param custom the custom data.
+   * @param description the human readable description of error.
+   */
+  public void log(Throwable error, Map<String, Object> custom, String description) {
+    log(error, custom, description, null);
+  }
+
+  /**
+   * Record a debugging message at the level returned by {@link Rollbar#level} (WARNING unless level
+   * is overridden).
+   *
+   * @param message the message.
+   */
+  public void log(String message) {
+    log(null, null, message, null);
+  }
+
+  /**
+   * Record a message with extra information attached at the default level returned by {@link
+   * Rollbar#level}, (WARNING unless level overridden).
+   *
+   * @param message the message.
+   * @param custom the extra information.
+   */
+  public void log(String message, Map<String, Object> custom) {
+    log(null, custom, message, null);
+  }
+
+  /**
+   * Record a message at the level specified.
+   *
+   * @param message the message.
+   * @param level the level.
+   */
+  public void log(String message, Level level) {
+    log(null, null, message, level);
+  }
+
+  /**
+   * Record a message with extra information attached at the specified level.
+   *
+   * @param message the message.
+   * @param custom the extra information.
+   * @param level the level.
+   */
+  public void log(String message, Map<String, Object> custom, Level level) {
+    log(null, custom, message, level);
+  }
+
+  /**
+   * Record an error or message with extra data at the level specified. At least ene of `error` or
+   * `description` must be non-null. If error is null, `description` will be sent as a message. If
+   * error is non-null, description will be sent as the description of the error. Custom data will
+   * be attached to message if the error is null. Custom data will extend whatever {@link
+   * Config#custom} returns.
+   *
+   * @param error the error (if any).
+   * @param custom the custom data (if any).
+   * @param description the description of the error, or the message to send.
+   * @param level the level to send it at.
+   */
+  public void log(Throwable error, Map<String, Object> custom, String description, Level level) {
+    process(error, custom, description, level);
+  }
+
+  /**
+   * Get the level of the error or message. By default: CRITICAL for {@link Error}, ERROR for other
+   * {@link Throwable}, WARNING for messages. Override to change this default.
+   *
+   * @param error the error.
+   * @return the level.
+   */
+  public Level level(Throwable error) {
```

Comment:
This appears to be unused

Reply:
That's right! I forgot to append the level information when building the data. I will commit a change. Thanks!